README.md

```diff
@@ -1,66 +1,55 @@
 
-Reproduce what applications are doing on the cluster for example DNS/UDP and HTTP/GRPC/TCP flow through the main network paths like internal, external ingresses and pod to pod communication through service ip.
+- Reproduce what applications are doing on the cluster for example DNS/UDP and HTTP/GRPC/TCP flow through the main network paths like pod to pod communication through service IPs.
 
-Exercise DNS/UDP network path (service ip CoreDNS) test-client-load-injector and test-backend-service resolving kubernetes and ingress hostname.
-
-Chaos testing in the background (resilient to failures):
-  - deleting test-backend-service (pod ip churn, graceful shutdown, recycle TCP connections: the load injector will be forced to create new connection)
-  - delete of nodes (exercising autoscaling or node termination)
-
-### Components
+## Components
 
 This project relies on: 
  - [k6](https://k6.io/) and statsd-exporter: acting as a load-injector
- - podinfo: [golang application](https://github.com/stefanprodan/podinfo), we are using a modified version available [here](https://github.com/rewiko/podinfo)
- - nginx ingress controller [link](https://github.com/kubernetes/ingress-nginx/)
- - istio [link](https://istio.io/latest/docs/)
- - monitoring stack [link](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
- - docker and [k3d](https://k3d.io/) to create local k8s cluster
-
-### Architecture 
-
-![Continuous Load](./docs/Continuous%20Load%20Diagram.png)
-
-The load-injector send load to source-podinfo and not to the ingress directly because load-injectors like k6 or gatling are design to retry on connection reset which can hide issues. It is also interresting to own a golang application to go further and add functional validation like JWT secret rotation, MTLS or talking to a database. 
+ - Podinfo: [golang application](https://github.com/stefanprodan/podinfo)
+ - Prometheus Operator [link](https://prometheus-operator.dev/)
+ - Grafana Operator [link](https://grafana-operator.github.io/grafana-operator/)
+ - Helm
 
 ### Monitoring
 
-Each component creates a ServiceMonitor or PodMonitor resource to configure prometheus automatically.
+A ServiceMonitor or PodMonitor resources are used to configure prometheus and are found in the relevant charts.
 
 ### Alerts
 
-Each component creates PrometheusRule resources to configure alerts on prometheus automatically.
-Those alerts follow the [Golden Signal principles](https://sysdig.com/blog/golden-signals-kubernetes/). 
+PrometheusRule resources are created to configure alerts on prometheus.
+Such alerts follow the [Golden Signal](https://sysdig.com/blog/golden-signals-kubernetes/) principles. 
 
 ### Dashboard 
 
-Continuous dashboard is create automatically and the link will be available within the console after running the script below.
+A GrafanaDashboard resource is used to create the continuous load dashboard.
 
-### Run 
+## Run 
```

Comment:
Can we change this section to "Running Locally"

Then add a new section about real life deployment e.g.


## Deployment

There are two helm charts:

* Prerequsites: everything needed to display the dashboards for Continuous Load
* Continuous Load: The run time components of Continuous Load

For an end to end demo, install both, for production deployments, consider integrating with an existing prometheus and grafana installation.


## Pre-configured monitoring stack (pre-requsites)

TODO show how to use the helm chart directly

## Continuous Load (no dashboarding)

TODO show how to use the helm chart directly

## Deploying dashboards

TODO show how to use an existing ns